src/fal/run_scripts.py

```diff
@@ -38,7 +38,7 @@ class Context:
     config: ContextConfig
 
 
-def run_scripts(list: List[FalScript], project: FalProject):
+def run_scripts(list: List[FalScript], project: FalProject, no_logging: bool):
```

Comment:
1) alright I think we should not even import the logger if this is disabled. 

2) Here is how we might do it; 

Create a new class called `FalLogger` import dbt.logger inside in a conditional with methods info, warn ... ect.. 

Pass around `Fallogger` everywhere, application code shouldnt care if logger is disabled or not all the conditional logic happens in the `Falloger` class